scripts/v.db.droprow/v.db.droprow.html

```diff
@@ -35,6 +35,7 @@ <h2>EXAMPLES</h2>
 <h2>SEE ALSO</h2>
 
 <em>
+<a href="db.vacuum.html">db.vacuum</a>,
```

Comment:
Also add this to
- db.dropcolumn.html
- db.dropdb.html
- db.droptable.html
?